css/style.css

```diff
@@ -763,3 +763,41 @@ html {
   margin : 10px;
   align-content: center;
 }
+body{
+	margin:0;
+	padding:0;
+	font-family:"arial",heletica,sans-serif;
+	font-size:12px;
+    background: #2980b9 url('http://static.tumblr.com/03fbbc566b081016810402488936fbae/pqpk3dn/MRSmlzpj3/tumblr_static_bg3.png') repeat 0 0;
+	-webkit-animation: 10s linear 0s normal none infinite animate;
```

Comment:
Please download the image, add it to `humblefool/img/` folder, and change the link here accordingly :)